README.md

```diff
@@ -9,7 +9,7 @@ A Minimal social networking application using Flask
     git clone https://github.com/SriNandan33/bootcamp.git 
     ```
 
-2. **Install Python version 3.6 or greater**
+2. **Install Python version 3.6**
```

Comment:
It works even with python 3.7 or above. So keep something like  `3.6 or above` , ` >=3.6` ...etc